pkg/ovs/ovsconfig/default.go

```diff
@@ -26,9 +26,13 @@ const (
 
 	defaultConnNetwork = "unix"
 	// Wait up to 5 seconds when getting port.
-	defaultGetPortTimeout = 5 * time.Second
+	defaultGetPortTimeout    = 5 * time.Second
+	defaultOvsVersionMessage = "ovs-vsctl --no-wait set Open_vSwitch . ovs_version=$OVS_VERSION"
```

Comment:
`ovs-vsctl --no-wait set Open_vSwitch . ovs_version=$OVS_VERSION` should not be used for Linux.

What about: "OVS version is not set in ovsdb"